inertia/tests/test_history.py

```diff
@@ -0,0 +1,32 @@
+from inertia.http import encrypt_history
+from inertia.test import InertiaTestCase
+from django.test import override_settings
+from django.test.client import RequestFactory
+from inertia.tests.testapp.views import empty_test
+
+
+class MiddlewareTestCase(InertiaTestCase):
```

Comment:
great test cases!

Reply:
Thanks! I noticed I didn't add any documentation. I have added some information and examples about this feature in the readme.